docs/releases/v1.1.0.rst

```diff
@@ -70,6 +70,12 @@
 
     -[enh] Added reducer functions :func:`dt.countna()` and :func:`dt.nunique()`. [#2999]
 
+    -[new] Class :class:`dt.FExpr` now has method :meth:`.nunique()`,
+      which behaves exactly as the equivalent base level function :func:`dt.nunique()`.
+
+    -[new] Class :class:`dt.FExpr` now has method :meth:`.countna()`,
+      which behaves exactly as the equivalent base level function :func:`dt.countna()`.
+
```

Comment:
Have you tried to build documentation? I doubt that  links to :meth:`.nunique()` and :meth:`.countna()` will actually point to `docs/api/fexpr/countna.rst` and `docs/api/fexpr/nunique.rst`.

Reply:
Nevermind, it will work since it it under the `FExpr` section.